src/math/mod.rs

```diff
@@ -15,10 +15,11 @@ mod sqrtf;
 mod logf;
 mod expf;
 mod floor;
+mod cosf;
 mod trunc;
 mod truncf;
 
-//mod service;
+mod service;
```

Comment:
(we don't *need* to put helpers in a module because we have the concept of privacy in Rust so these helpers could be files in src/math.rs -- the important part is not make them public when we re-export them in the math module (i.e. use `use ..` or maybe `pub(crate) use ..` instead of `pub use ..`)